src/html/index.html

```diff
@@ -0,0 +1,71 @@
+    <img src="images/more-pills.svg" alt="Three pill tablets." id="more-pills">
+    <img src="images/purple-pills.svg" alt="Two purple pills." id="purple-pills">
+  </div>
+    <nav class="pillbar">
+      <ul>
+        <a href="#box-one"><li class="left-end active">The roblem</li></a>
+        <a href="#key-factors" onclick="scroll(1)"><li>Key Factors</li></a>
+        <a href="#questions"><li>The Question</li></a>
+        <a href="#solution"><li class="right-end ">The Solution</li></a>
+      </ul>
+    </nav>
+
+    <div class="box" id="box-one">
+      <p id="main-thought">Patients who don’t take medication as prescribed are at risk of worsening health, increased hospitalization, and a greater chance of mortality.</p>
+      <p>Just in the U.S., non-adherence has been attributed to over $100 billion dollars in avoidable healthcare costs.<sup>1</sup> In developed countries, among patients with chronic illness, approximately 50% do not take medications as prescribed.<sup>2</sup></p>
+
+      <div class="quote">
+        <p id="quote-text">“Increasing the effectiveness of adherence interventions might have a greater impact on the health of the population than any improvement in specific medical treatment.”</p>
+        <p class="attribution">World Health Organization, 2005 <sup>3</sup><p>
+      </div>
+      <img src="images/pill-bottle.svg" alt="Short yellow pill bottle." id="pill-bottle">
+      <img src="images/pill-bottle2.svg" alt="Tall yellow pill bottle." id="pill-bottle2">
+      <img src="images/pill-bottle3.svg" alt="Red medicine bottle." id="pill-bottle3">
+    </div>
+
+    <div id="key-factors">
+      <h2 class="section-title">Why People Don't Adhere</h2>
+      <p class="section-intro">A myriad of factors conspire to prevent people from sticking to their treatment programs. But a few are most telling.</p>
+      <img src="images/body.svg" alt="Simple human body." id="body">
+      <div class="factor-box factor-box-right" id="comprehension">
+        <img src="images/gears.svg" alt="Yellow gears icon.">
+        <h3>Comprehension</h3>
+        <p>“All of that medical jargon doesn’t make much sense to me but I don’t want to bother the doctor with my silly questions.”</p>
+      </div>
+      <div class="factor-box factor-box-left" id="denial">
+        <img src="images/heart.svg" alt="Heart icon.">
+        <h3>Denial</h3>
+        <p>“I feel just fine. I don’t know why that doctor thinks I’m sick. Why would I take medicine that I don’t need?”</p>
+      </div>
+      <div class="factor-box factor-box-right" id="cost">
+        <img src="images/money.svg" alt="Money icon.">
+        <h3>Cost</h3>
+        <p>“I took my prescription to the pharmacy but they said it would cost several hundred dollars! I just couldn’t afford it.”</p>
+      </div>
+      <div class="factor-box factor-box-left" id="complexity">
+        <img src="images/tally.svg" alt="Money icon.">
```

Comment:
These circle icons would be good candidates for pseudo elements in css, e.g.
```
.factor-box {
  &:before {
    content: '';
    display: block;
    width: 40px;
    height: 40px;
    position: absolute;
    top: 0;
    bottom: 0;
    margin: auto;
  }

  &-left {
    right: 0;
  }

  &-right {
    left: 0;
  }
}
```

You'd then just have to key off the box id (or class if you switch to classes [for specificity reasons]) to get a different background on each one.

```
.factor-box {
  &-complexity {
    background-image: url(...);
  }
}
```